plugins/pulp_rpm/plugins/distributors/yum/configuration.py

```diff
@@ -3,8 +3,9 @@
 from ConfigParser import SafeConfigParser
 from gettext import gettext as _
 
+from pulp.server.db import model
+from pulp.server.controllers import distributor as dist_controller
 from pulp.server.exceptions import MissingResource
-from pulp.server.managers import factory
```

Comment:
The more diffs we have that remove this line, the more Pulp approaches being a respectable Python project. +1